fix: make port 22 in transparent-proxy clear (#1490)
kumahq/kuma-website: app/_src/production/dp-config/transparent-proxying.md

     --redirect-dns
   ```

> 

Reviewer: nitpick:

```suggestion
```

{% warning %}
Please note that this command **will change** the host `iptables` rules.
It's recommended to install the transparent proxy in a virtual network to easily recreate things from scratch.
{% endwarning %}

> 

Reviewer: nitpick:

```suggestion
```

> The changes will persist over restarts, so this command is needed only once. Reverting to the original state of the host can be done by issuing `kumactl uninstall transparent-proxy` {% if_version gte:2.2.x %} (which is not implemented yet see issue [#8071](https://github.com/kumahq/kuma/issues/8071)).{% endif_version %}

Reviewer: It only makes sense when you know that we rewrote TP. Why would we mention a command that does not exist.

I'd go with:

```suggestion
The changes will persist over restarts, so this command is needed only once. {% if_version lte:2.1.x %} Reverting to the original state of the host can be done by issuing `kumactl uninstall transparent-proxy`.{% endif_version %}{% if_version gte:2.2.x %} Reverting to the original state of the host is not yet implemented, see issue [#8071](https://github.com/kumahq/kuma/issues/8071).{% endif_version %}
```

Reviewer 2: Wait.`kumactl install transparent-proxy` needs to be executed after every system reboot or you need to have some solution for iptables rules persistence in place. Otherwise, the transparent proxy configuration won't survive restarts.

Reviewer 2: Wait. `kumactl install transparent-proxy` needs to be executed after every system reboot or you need to have some solution for iptables rules persistence in place. Otherwise, the transparent proxy configuration won't survive restarts.
